movepedia

```diff
@@ -1,9 +1,14 @@
 #! /usr/bin/env php
```

Comment:
Why is this here?

Reply:
It is required for auto-detection of the type of script